src/utils/decorators/input-validation/input-validation.js

```diff
@@ -518,9 +591,15 @@ let InputValidation = (ComposedComponent) => class Component extends ComposedCom
    * @return {HTML} Validation HTML including icon & message
    */
   get validationHTML() {
-    if (this.state.valid && !this.state.warning) { return null; }
+    let type = '';
+
+    if (this.state.valid && !this.state.warning && !this.state.info) { return null; }
 
-    let type = !this.state.valid ? "error" : "warning";
+    if (this.state.info) {
+      type = "info";
```

Comment:
Just to confirm, we are stating that the order of importance is `info` then `error` then `warning`? May need to confirm this with UX.